source/common/http/access_log/access_log_formatter.cc

```diff
@@ -17,39 +17,73 @@ const std::string FilterReasonUtils::UPSTREAM_CONNECTION_TERMINATION = "UC";
 const std::string FilterReasonUtils::UPSTREAM_OVERFLOW = "UO";
 const std::string FilterReasonUtils::NO_ROUTE_FOUND = "NR";
 const std::string FilterReasonUtils::FAULT_INJECTED = "FI";
+const std::string FilterReasonUtils::DELAY_INJECTED = "DI";
```

Comment:
Delete until this is actually used in the code, or use it in the code, and also add to docs.